lte/gateway/python/magma/pipelined/app/gy.py

```diff
@@ -283,7 +283,7 @@ def _get_rule_match_flow_msgs(self, imsi, msisdn: bytes, uplink_tunnel: int, ip_
                     imsi, msisdn, uplink_tunnel, ip_addr, apn_ambr, flow, rule_num, priority,
                     rule.qos, rule.hard_timeout, rule.id, rule.app_name,
                     rule.app_service_type, self.next_service_table,
-                    version, self._qos_mgr, self._enforcement_stats_tbl))
+                    version, self._qos_mgr, shard_id, self._enforcement_stats_tbl))
```

Comment:
ah I hate how long cwf integ runs... the issue is in this function call, shard_id should be the last argument